src/Confluent.Kafka/KafkaException.cs

```diff
@@ -23,12 +23,19 @@ namespace Confluent.Kafka
 {
     public class KafkaException : Exception
     {
-        public KafkaException(Error error, string message)
-            : base(message)
+        private Error _error;
```

Comment:
Isnt having the same name on the member as its type more confusing than the uglyness of underscore prefix?